terms/r1cs.md

```diff
@@ -0,0 +1,37 @@
+# R1CS
+
+## R1CS
+
+The R1CS structure consists of spare matrices $A, B, C$. An R1CS instance
+consists of public inputs and outputs $x$ and is satisfied by a witness $W$ if:
```

Comment:
```suggestion
consists of public (inputs, outputs) $x$ and is satisfied by a witness $W$ if:
```
I think this is more clear

Reply:
In the definition of the paper, the author used what I wrote